### PR TITLE
Bump to & release v0.1.0

### PR DESCRIPTION
**Pull request by andyscott**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,7 +157,7 @@ val evalAlgebra2: Algebra ~> Future = CopK.FunctionK.summon
 ```
 
 The interpreters created by Iota are optimized for speed and have a
-constant evaluation time. Behind the scenes, a macro generates an
+constant evalaluation time. Behind the scenes, a macro generates an
```
Comment (peterneyens) [inline README.md]: I don't think `evalaluation` is an improvement over `evaluation` :smiley:

File: README.md [hunk 2]
```diff
@@ -39,13 +38,13 @@ For Scala 2.11.x and 2.12.x:
 [comment]: # (Start Replace)
 
 ```scala
-libraryDependencies += "com.47deg" %% "iota-core" % "0.0.2"
+libraryDependencies += "com.47deg" %% "iota-core" % "0.1.0"
```
Comment (juanpedromoreno) [inline README.md]: You don't need to do this, sbt-org-policies will replace the version automatically.

File: README.md [hunk 3]
```diff
@@ -39,13 +38,13 @@ For Scala 2.11.x and 2.12.x:
...
 ```
 
 Or, if using Scala.js (0.6.x):
 
 ```scala
-libraryDependencies += "com.47deg" %%% "iota-core" % "0.0.2"
+libraryDependencies += "com.47deg" %%% "iota-core" % "0.1.0"
```
Comment (juanpedromoreno) [inline README.md]: Same here.

File: modules/readme/src/main/tut/README.md [hunk 4]
```diff
@@ -1,4 +1,3 @@
-
```
Comment (juanpedromoreno) [inline modules/readme/src/main/tut/README.md]: Same as here https://github.com/47deg/iota/pull/27/files#r115976579

File: modules/readme/src/main/tut/README.md [hunk 5]
```diff
@@ -39,13 +38,13 @@ For Scala 2.11.x and 2.12.x:
 [comment]: # (Start Replace)
 
 ```scala
-libraryDependencies += "com.47deg" %% "iota-core" % "0.0.2"
+libraryDependencies += "com.47deg" %% "iota-core" % "0.1.0"
```
Comment (juanpedromoreno) [inline modules/readme/src/main/tut/README.md]: You don't need to bump the version here.

File: README.md (hunk 2 above)
Comment (juanpedromoreno) [inline README.md]: BTW, the README file in the root is also automatically updated by `tut` and commited by sbt-org-policies when the one in the tut sources is compiled.

File: README.md (hunk 1 above)
Comment (andyscott) [inline README.md]: What about evalalalauation? @MaureenElsberry 😄

**Comment by andyscott** [comment]
@juanpedromoreno is it 👍 now?

**Pull request merged**